src/vim9class.c

```diff
@@ -1784,6 +1784,76 @@ ex_type(exarg_T *eap UNUSED)
     // TODO
 }
 
+/*
+ * Returns OK if a member variable named "name" is present in the class "cl";
+ * otherwise returns FAIL.  If found, indirectly set the return values,
+ * including the class where it is defined.
+ *
+ * NOTE: set internal flag "check_super" to TRUE
+ *	 to return the class where name is defined.
+ * NOTE: this may need adjustment when class member inheritance is changed.
+ */
+// TODO: fold this into the one place it is used, unless there might
+//	 be another use for it.
+    static int
+find_class_member_definition(
+    class_T	*cl,
+    char_u	*name,
+    size_t	namelen,
+    ocmember_T	**ret_m,
+    class_T	**ret_cl)
+{
+    int		check_super = FALSE; // If statics are shared, not inherited,
```

Comment:
Once we decide static variables are shared or inherited, then we don't need the `check_super` variable?

Reply:
I think that's true.